src/Avalonia.Input/TouchDevice.cs

```diff
@@ -0,0 +1,70 @@
+using System.Collections.Generic;
+using System.Linq;
+using Avalonia.Input.Raw;
+using Avalonia.VisualTree;
+
+namespace Avalonia.Input
+{
+    /// <summary>
+    /// Handles raw touch events
+    /// This class is supposed to be used on per-toplevel basis, don't event try to have a global instance
```

Comment:
Nit: should be "don't _even_". Also this should be in `<remarks>` and maybe explain why?